From 192e5b32af1514dcc398e37739a1b1aca94ab9d7 Mon Sep 17 00:00:00 2001
From: Nathan Froyd <froydnj@mozilla.com>
Date: Thu, 2 Jan 2020 15:38:20 +0000
Subject: [PATCH 25/35] Bug 1606625 - don't add PGO flags for wasm compilation;
 r=rstewart

At the moment, everything we're putting in a wasm sandbox is not
performance-critical, so we don't need PGO.  It's also not clear that
PGO would actually work properly with code that's been run through
wasm.  Let's leave figuring that out until, at the very least, the wasm
toolchain is a little more mature.

Differential Revision: https://phabricator.services.mozilla.com/D58513

--HG--
extra : moz-landing-system : lando

diff --git a/config/config.mk b/config/config.mk
index 98fd0e093613..efe180aa6ea3 100644
--- a/config/config.mk
+++ b/config/config.mk
@@ -194,8 +194,8 @@ HOST_CXXFLAGS = $(COMPUTED_HOST_CXXFLAGS) $(_DEPEND_CFLAGS)
 HOST_C_LDFLAGS = $(COMPUTED_HOST_C_LDFLAGS)
 HOST_CXX_LDFLAGS = $(COMPUTED_HOST_CXX_LDFLAGS)
 
-WASM_CFLAGS = $(COMPUTED_WASM_CFLAGS) $(PGO_CFLAGS) $(_DEPEND_CFLAGS) $(MK_COMPILE_DEFINES)
-WASM_CXXFLAGS = $(COMPUTED_WASM_CXXFLAGS) $(PGO_CFLAGS) $(_DEPEND_CFLAGS) $(MK_COMPILE_DEFINES)
+WASM_CFLAGS = $(COMPUTED_WASM_CFLAGS) $(_DEPEND_CFLAGS) $(MK_COMPILE_DEFINES)
+WASM_CXXFLAGS = $(COMPUTED_WASM_CXXFLAGS) $(_DEPEND_CFLAGS) $(MK_COMPILE_DEFINES)
 WASM_LDFLAGS = $(COMPUTED_WASM_LDFLAGS)
 
 ifdef MOZ_LTO
-- 
2.25.1

